scripts/common/rook.sh

```diff
@@ -117,8 +135,8 @@ function remove_rook_ceph() {
     fi
 
     # print success message
-    printf "%bRemoved rook-ceph successfully!\n%b" "$GREEN" "$NC"
-    printf "Data within /var/lib/rook, /opt/replicated/rook and any bound disks has not been freed.\n"
+    logSucces "Removed rook-ceph successfully!"
+    logWarn  "Data within /var/lib/rook, /opt/replicated/rook and any bound disks has not been freed."
```

Comment:
@rrpolanco @ricardomaraschini, @emosbaugh @laverya 

Why we print that? 
I could not find another place where it is done.
So, why we do not cleanup the any bound disks?
Do you know that?